### ISSUE TITLE
This causes the edit page to shake

### ISSUE THREAD

**Issue by ShadylHl** [ckRobinson/multi-column-markdown#89]
The preamble reference to the column, when I continue to edit the text, the editing page will shake, which is very uncomfortable; When I unblock the column code, everything becomes normal。

**Comment by eofferma** [comment]
I discover that (at least in my case) it happens when I click the note.

![Peek 2023-09-22 08-45](https://github.com/ckRobinson/multi-column-markdown/assets/475810/3a75e93f-e2b6-4a24-bd6b-595724a4b5e1)

**Comment by cday960** [comment]
#87

**Comment by ckRobinson** [comment]
This issue has been fixed in an upcoming release, hope to get it out soon(*tm*) but time is very limited for me right now.

**Comment by ckRobinson** [comment]
Hi all, version 0.9.0 with a fix for this issue has been released for just over a week now and I will be closing this issue, feel free to open another issue if you see something else and I'll try to get to it when I can, thanks.
